src/components/seat.tsx

```diff
@@ -0,0 +1,20 @@
+import { Checkbox } from "@headlessui/react";
+
+type Props = {
```

Comment:
Make disabled optional and give it a default value of false.
You don't want to pass disabled at all if it's false